scripts/log2dch

```diff
@@ -112,6 +114,11 @@ print_hackmd() {
     if [[ $buf == 'tests:'* ]]; then
 	buf="~~NOTEST ${buf}~~"
     fi
+    for filter in $FILTER_LIST; do
```

Comment:
You defined FILTER_LIST as an array, so you need to use it as one:

```
local filter=""
for filter in ${FILTER_LIST[@]); do 
    ....
done
```

Reply:
`"${FILTER_LIST[@]}"`

^-- quote it. otherwise IFS still applies.